bin/compile

```diff
@@ -17,30 +27,12 @@ function indent() {
 
 echo '-----> Installing binary dependencies for ActiveStorage Preview'
 
-SFFMPEG_VERSION="4.2.3"
-
-# Store which ffmpeg version was installed in the cache to bust the cache if it changes
-if [ -f $CACHE_DIR/.apt/SFFMPEG ]; then
-	CACHED_SFFMPEG=$(cat "$CACHE_DIR/.apt/SFFMPEG")
-else
-	CACHED_SFFMPEG=$SFFMPEG_VERSION
-fi
+SFFMPEG_VERSION="5.1.2"
```

Comment:
Can we rename this to `FFMPEG_VERSION` please? :)

Reply:
Updated, check and resolve please.